synapse/config/registration.py

```diff
@@ -333,24 +333,6 @@ def generate_config_section(self, generate_secrets=False, **kwargs):
         #
         #default_identity_server: https://matrix.org
 
-        # The list of identity servers trusted to verify third party
-        # identifiers by this server.
-        #
-        # Also defines the ID server which will be called when an account is
-        # deactivated (one will be picked arbitrarily).
-        #
-        # Note: This option is deprecated. Since v0.99.4, Synapse has tracked which identity
-        # server a 3PID has been bound to. For 3PIDs bound before then, Synapse runs a
-        # background migration script, informing itself that the identity server all of its
-        # 3PIDs have been bound to is likely one of the below.
-        #
-        # As of Synapse v1.4.0, all other functionality of this option has been deprecated, and
-        # it is now solely used for the purposes of the background migration script, and can be
```

Comment:
Should we remove the code surrounding this too (e.g. the background script)?

Reply:
It’s probably still too soon to do that since that would break it for anyone updating from before 1.4.0 to the current version. This would just remove that option from being inserted into newly generated `homeserver.yaml` files which I think is safe to do.

Reply:
Sounds good. 👍